docs/imports.md

```diff
@@ -8,7 +8,7 @@ import cats.data._
 import cats.syntax.all._
 ```
 
-The `cats._` import brings in quite a few [type classes](typeclasses.md) (similar to interfaces) such as [Monad](typeclasses/monad.md), [Semigroup](typeclasses/semigroup.md), and [Foldable](typeclasses/foldable.md). Instead of the entire `cats` package, you can import only the types that you need, for example:
```

Comment:
prefer to use `cats.*` as it is backport to scala 2, and underscore import is deprecated in scala 3. Similar in other places.

Reply:
There is one existing usage of a `.*` import in the docs. If we change it, I would much rather see it done as a separate PR doing the replacement everywhere.